wallet-api/src/Wallet/API.hs

```diff
@@ -61,6 +61,7 @@ data EventTrigger =
 data WalletAPIError =
     InsufficientFunds Text
     | OtherError Text
+    deriving (Show)
```

Comment:
Is this for testing?

Reply:
it's so we can send the error over HTTP easily